### PR TITLE
[343] feat: update landing page into 2023

### PR DESCRIPTION
**Pull request by yungshenglu**
## Types of changes

* **New feature**

## Description

- Update our landing page into 2023 designs
- [Desktop](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=5132%3A45173&t=W0YV315qZnD8bftU-0)
- [Mobile](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=5132%3A40172&t=W0YV315qZnD8bftU-0)

## Expected behavior

https://user-images.githubusercontent.com/11289171/207521035-5897dec8-c817-4729-bb72-734e3df77147.mov

## Related Issue
#343 

## Additional context
NA

### CODE REVIEW

**Comment by josix** [comment]
@yungshenglu It appears that there is a conflict with the configuration for displaying buttons on the index page (reference #351). Could you please help to investigate and resolve the issue?

File: pages/index.vue [hunk 1]
```diff
@@ -159,81 +140,64 @@ export default {
...
+        background-image: url('~@/static/landing-background.svg');
+        background-size: cover;
+        background-position-y: 0;
+    }
+    .landing__title {
+        background-image: url('~@/static/landing-title.svg');
+        background-repeat: no-repeat;
+        background-position-x: 0;
+        background-position-y: 2.5rem;
+        background-size: 60%;
```
Comment (josix) [inline pages/index.vue:184]: **issue (blocking):** looks like the title image is not displaying correctly on my devices with screen resolutions of 1920x1080 and 1080x800, as shown in the following images.

**1920x1080**
image

**1080x800**
image

File: config/pageLanding.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export const landingButtonConfig = {
```
Comment (josix) [inline config/pageLanding.js:1]: **praise:** Niiice extraction!

**Review by josix — CHANGES_REQUESTED** [review]
Thanks @yungshenglu, I've left some primary issues first. PTAL

**Review by josix — CHANGES_REQUESTED** [review]
nit. I guess the following section could be removed, WDYT?
```js
        isCallingForProposals() {
            return this.$store.state.configs.showSpeakingPage
        },
        isShowingScheduleButton() {
            return this.$store.state.configs.showSchedulePage
        },
        showIndexSponsorSection() {
            return this.$store.state.configs.showIndexSponsorSection
        },
        showIndexAPACSection() {
            return this.$store.state.configs.showIndexAPACSection
        },
```

**Review by josix — APPROVED** [review]

**Comment by yungshenglu** [comment]
> ```js
> isCallingForProposals
> ```

I need to check

**Comment by anan0202** [comment]
Web banner png and svg file has already been update. You can find the "Website banner" in the link below. Thank you very much. You are the best engineer!

https://drive.google.com/drive/u/0/folders/1bx-JsEhpnkfo85JD-cSvk4Swe6_5XgtO

**Comment by yungshenglu** [comment]
> Web banner png and svg file has already been update. You can find the "Website banner" in the link below. Thank you very much. You are the best engineer!
> 
> https://drive.google.com/drive/u/0/folders/1bx-JsEhpnkfo85JD-cSvk4Swe6_5XgtO

Updated.

**Review by josix — APPROVED** [review]
LGTM!

**Review by anan0202 — APPROVED** [review]
Awesome! 太 Awesome 拉！

**Pull request merged**